src/main/java/cz/smarteon/loxone/LoxoneAuth.java

```diff
@@ -427,4 +434,12 @@ private boolean isNewSaltNeeded() {
         }
         return false;
     }
+
+    private <T> T onVisuPassSet(String actionDescription, Supplier<T> action) {
+        if (loxoneVisPass != null) {
+            return action.get();
+        } else {
+            throw new IllegalStateException("Can't " + actionDescription + " when visualization password not set.");
+        }
+    }
 }
```

Comment:
Pls add missing newline at the end of file.

Reply:
fixed